README.md

```diff
@@ -111,6 +111,7 @@ There’s also special non-standard methods that aren’t included within the or
 * `evm_revert` : Revert the state of the blockchain to a previous snapshot. Takes a single parameter, which is the snapshot id to revert to. If no snapshot id is passed it will revert to the latest snapshot. Returns `true`.
 * `evm_increaseTime` : Jump forward in time. Takes one parameter, which is the amount of time to increase in seconds. Returns the total time adjustment, in seconds.
 * `evm_mine` : Force a block to be mined. Takes no parameters. Mines a block independent of whether or not mining is started or stopped.
+* `evm_mineAtTimestamp` : Force a block to be mined exactly at the given timestamp. Takes one parameter which is the expected timestamp of a new block in seconds.
```

Comment:
This is confusing to me. Reading this, I'd expect this not to instamine, but rather to wait until `now() >= timestamp` (likely via a `setTimeout`) and mine a block then, with whatever transactions are pending at the time.

Also, I think we are cluttering the `evm_` RPC namespace here. Since this is clearly a testing facility, perhaps we should use this as an opportunity to introduce a `testing_` namespace? I'm on the fence about this, however...

Reply:
I've updated the description. I hope it's clear now. 
The main reason to have a separate function was to maintain backward compatibility, but I totally agree that I'd be way better to have a more concise api.  So, I followed a pattern you're using in different functions with optional argument repositioning. Let me know if it looks good for you.